client/components/Averages.jsx

```diff
@@ -1,9 +1,27 @@
 import React from 'react';
+import styled from 'styled-components';
+import LabelledBar from './LabelledBar.jsx';
 
-var Averages = (props) => (
-  <div>
+var Styled = styled.div`
```

Comment:
You probably have thought of this, but I started putting Styles in their own separate file and it makes components look clean.

Reply:
Yeah, good point. I might refactor later.